scripts/mergedPRDetailsToReleaseDraft.ps1

```diff
@@ -0,0 +1,83 @@
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [string]$CommitMessage,
+    [string]$Token
+)
+
+$header = @{
+    "authorization" = "token $Token"
+    "Accept"        = "application/vnd.github.v3+json"
+}
+
+#Get all releases including drafts
+$getReleases = Invoke-RestMethod -Method Get -Headers $header -URI  "https://api.github.com/repos/StefanIvemo/ActionsTest/releases"
```

Comment:
This URI points to a different repo than this. is this correct?

Reply:
😅 Ooops!

Reply:
Looks better, but I would instead suggest setting the URI as a variable and referencing that one. That way, if the module ever changes home it's way easier to change this as well.

Reply:
I'll fix